opg-base-1604/README.md

```diff
@@ -53,8 +53,9 @@ Supported variables
 -------------------
 * OPG_DOCKER_TAG - see versions
 * SKIP_SSL_GENERATE - when set container will not create self signed certificate on start
-* OPG_BASE_SSL_CERT - pass a cert as a multiline or using `"\n"`
-* OPG_BASE_SSL_KEY - pass a key as a multiline or using `"\n"`
+* SSL_CERT_COMMON_NAME - if set, its value will be used for `common name` while generating the self signed certificate 
+* OPG_BASE_SSL_CERT - pass a cert as a multiline or using
```

Comment:
You seem to have lost the `"\n"` portion in these two strings

Reply:
put back